ptzclient/README.md

```diff
@@ -0,0 +1,131 @@
+# PTZ model
+
+This model implements the [`"rdk:component:generic"` API](https://docs.viam.com/components/generic/) for controlling ONVIF-compliant PTZ (Pan-Tilt-Zoom) cameras. The generic component supports core PTZ operations through the DoCommand method.
+
+## Configure your `onvif-ptz-client`
+
+1. Add the `onvif-ptz-client` generic component
+2. Configure connection parameters and profile token:
+
+```json
+{
+  "address": "192.168.1.100:80",
+  "username": "admin",
+  "password": "yourpassword",
+  "profile_token": "000"
+}
+```
+
+### Attributes
+
+| Name | Type | Inclusion | Description |
+|------|------|-----------|-------------|
+| `address` | string | **Required** | Camera IP address with port |
+| `username` | string | **Required** | ONVIF authentication username |
+| `password` | string | **Required** | ONVIF authentication password |
+| `profile_token` | string | Optional | Media profile token for PTZ control (discover with `get-profiles` command) |
+
+### Example Configuration
+
+```json
+{
+  "model": "viam:viamrtsp:onvif-ptz-client",
+  "attributes": {
+    "address": "192.168.1.100:80",
+    "username": "admin",
+    "password": "yourpassword",
+    "profile_token": "001"
+  }
```

Comment:
this is incomlete. Please provide the full config,

e.g. api is missing

Reply:
name is also missing

Reply:
Provided.